src/main/java-templates/org/postgresql/Driver.java

```diff
@@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
      * @throws SQLException if the connection could not be made
      */
     private static Connection makeConnection(String url, Properties props) throws SQLException {
-        return new @JDBCCONNECTCLASS@(hostSpecs(props),
+        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
```

Comment:
@mkarg This method returns a different instance depending on the version of JDBC we are building for. It can be either:
- Jdbc42Connection
- Jdbc4Connection

We need a performant way of achieving the same result. We could use a properties file that is embedded in the jar containing the major and minor versions as well as the spec version and use that to instantiate the correct class.